src/plugins/discourse/update_discourse_api_snapshots.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eu
+
+snapshots_dir=src/plugins/discourse/snapshots
+test_instance_url="https://sourcecred-test.discourse.group"
+test_instance_username="credbot"
+
+if [ -z "${DISCOURSE_TEST_API_KEY:-}" ]; then
+  printf >&2 'Please set the DISCOURSE_TEST_API_KEY environment variable.\n'
+  printf >&2 'Contact the SourceCred maintainers to get the key.\n'
+  return 1
```

Comment:
This will fail—you can only `return` from a function:

```
Please set the DISCOURSE_TEST_API_KEY environment variable.
Contact the SourceCred maintainers to get the key.
./test.sh: line 11: return: can only `return' from a function or sourced script
```

Try `exit 1`, or wrap the whole thing in a `main` function.